docs/technical-reference/documentation-contributions.md

```diff
@@ -4,34 +4,43 @@ title: Documentation contributions
 sidebar_label: Documentation contributions
 ---
 
-We use [Docusaurus](https://docusaurus.io/) for our docs. The repository for those docs is [OpenRefine/openrefine.org](https://github.com/OpenRefine/openrefine.org). For small documentation changes, you should be able to edit the Markdown files directly and submit them as a pull request. Each documentation page has a link at the bottom to help you do this in a few clicks. Once you submit your pull request, a preview of the docs will be generated automatically. 
+We welcome contributions to our documentation, which is maintained as a set of [Markdown](https://www.markdownguide.org/) files in the [OpenRefine/openrefine.org](https://github.com/OpenRefine/openrefine.org) Git repository.
 
```

Comment:
Should we divide the page into two sections
1. Single page edit via GitHub web interface 
2. Multi-page edit by editing the documentation locally

Reply:
Isn't it already the case?

Reply:
Sorry if my comment was not clear. My suggestion is to add a new h2 title "single page edit" and rename the other h2.

Reply:
Ok, I see! Perhaps it's simple enough if you edit the PR directly as you see fit?